src/write_gcm.c

```diff
@@ -2893,7 +2893,7 @@ static int wg_json_CreateCollectdTimeseriesRequest(
     const wg_payload_t *head, const wg_payload_t **new_head, char **json)
 {
   char name[256];
-  int result = snprintf(name, sizeof(name), "project/%s",
+  int result = snprintf(name, sizeof(name), "projects/%s",
```

Comment:
Can we not just remove this and the "name" field in [lines 2910­--2912](https://github.com/Stackdriver/collectd/pull/171/files#diff-f54cd6830a1d91892682e12d794bde80R2910-R2912)?

Reply:
Yes, as discussed offline, the backend takes the project ID from the request URL, not from this field.